lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -520,6 +525,11 @@ class Table(Entity):
             ]
         ],
     )
+    last_updated: Optional[datetime] = Field(
```

Comment:
I've got this the other way round - I've used the existing last_modified for the data last updated. 😕 

What do you think about renaming these fields to `data_last_updated` and `metadata_last_updated`?